renovate.json

```diff
@@ -17,32 +12,16 @@
     {
       "groupName": "all dependencies",
       "groupSlug": "all",
-      "matchBaseBranches": ["main"],
-      "enabled": false,
-      "matchPackagePatterns": [
-        "*"
-      ]
-    }, {
       "enabled": false,
```

Comment:
Updates are not enabled?

Reply:
No, RH instance does not support golang (they removed the go support few months ago after we introduced it in our projects). It was not working properly and the updates were not correctly composed.

Reply:
Are there plans to switch to another renovate instance?

Reply:
no